sim/adversary/repeat.go

```diff
@@ -100,7 +100,7 @@ func (r *Repeat) ReceiveMessage(msg *gpbft.GMessage, _ bool) (bool, error) {
 		Ticket:        ticket,
 	}
 	for i := 0; i < echoCount; i++ {
-		r.host.Broadcast(echo)
+		r.host.Broadcast(echo.Sender, echo)
```

Comment:
This can just use the normal `RequestBroadcast` method. It doesn't need to pass the sender explicitly.